src/provider/view/components/NotificationCard/Loading.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
Why do we need a loading screen? Is this ever used?

Reply:
`Loading.ts` and the [[loading, setLoading]](https://github.com/HadoukenIO/notifications-service/pull/105#discussion_r325659260) you commented on before lay the groundwork for [SERVICE-605](https://appoji.jira.com/browse/SERVICE-605). So who ever works on that story can just simply flip the state once they have the provider back-end logic working. Can add a `TODO: SERVICE-623 ...` comment so its more clear?